Formula/zig.rb

```diff
@@ -0,0 +1,34 @@
+class Zig < Formula
+  desc "Programming language designed for robustness, optimality, and clarity"
+  homepage "http://ziglang.org/"
+  url "https://s3.amazonaws.com/ziglang.org/download/0.1.1/zig-0.1.1.tar.xz"
+  sha256 "a160d14aecebead2f8c574b0dd687cdfecc53d149d521a17b89b629d8b558c94"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@5" => :build
+
+  def install
+    Dir.mkdir "build"
+    Dir.chdir "build" do
+      system "cmake", "..",
+        "-DCMAKE_PREFIX_PATH=#{prefix}/llvm@5",
+        "-DCMAKE_INSTALL_PREFIX=#{prefix}"
+      system "make", "install"
+    end
+  end
+
+  test do
+    src = <<-HEREDOC
+
+      const io = @import("std").io;
+
+      pub fn main() -> %void {
+          %%io.stdout.printf("Hello, world!");
+      }
+
+    HEREDOC
+    File.write("hello.zig", src)
+    system "#{bin}/zig", "build-exe", "hello.zig"
+    system "./hello"
```

Comment:
Can you check that the output from `hello` is as expected?
`assert_equal "Hello, world!\n", shell_output("./hello")`

Reply:
Yes, that's much better thanks.